frontend/common/src/components/UserProfile/ExperienceAccordion/individualExperienceAccordions/CommunityAccordion.tsx

```diff
@@ -34,6 +34,7 @@ const CommunityAccordion: React.FunctionComponent<CommunityAccordionProps> = ({
           <li>
             <p data-h2-font-color="b(lightpurple)">{skill.name?.[locale]}</p>
             <p>{skill.description?.[locale]}</p>
+            <p>{skill.experienceSkillRecord?.details}</p>
```

Comment:
This may lead to rendering an empty `<p>` tag. We should check for the existence before rendering the `<p>` tag.

Reply:
[here](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3460/commits/4c57deae82f13b1e972f46af3300bfdc5d0e4db0)